frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -5,15 +5,13 @@
 
 # Common
 
-action.GitMachete.description.disabled.undefined.branch-name={0} disabled due to undefined branch name
-action.GitMachete.description.disabled.undefined.machete-branch={0} disabled due to undefined machete branch
+action.GitMachete.description.disabled.undefined.machete-branch={0} disabled for {1} branch since it''s not managed by Git Machete
 action.GitMachete.description.disabled.branch-is-root={0} disabled because the given machete branch is a root
 
 action.GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec <b>{0}</b> failed
 action.GitMachete.FetchBackgroundable.notification.success=Fetch of refspec <b>{0}</b> succeeded
 
 action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.notification.message.view-commits=View Commits
-action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.notification.title.all-files-are-up-to-date=All files are up-to-date
```

Comment:
Whoops okay must be restored, it is used indeed (but broken with `+` hence IntelliJ couldn't find it)

Reply:
Yeah, I joined it (despite it's quite long)